config/application.rb

```diff
@@ -90,5 +90,8 @@ class Application < Rails::Application
     #   is_test                   - (NOT advisable because test plans are excluded from statistics)
     #   privately_visible         - Only the owner and people they invite can access the plan
     config.default_plan_visibility = 'privately_visible'
+   
+    # The percentage of answered questions needed to enable the plan visibility section of the Share plan page
+    config.default_plan_percentage_answered = 50.00
```

Comment:
I would put that constant under config/initializers/constant.rb and I would leave application.rb for rails specific.

Reply:
ah. agreed. I had forgotten about that file. I'll make that adjustment and send along in my next PR